include/wx/osx/cocoa/private.h

```diff
@@ -341,6 +341,8 @@ public :
     CGWindowLevel   GetWindowLevel() const wxOVERRIDE { return m_macWindowLevel; }
     void            RestoreWindowLevel() wxOVERRIDE;
 
+    bool m_macIgnoreNextFullscreenChange;
```

Comment:
```suggestion
    bool m_macIgnoreNextFullscreenChange = false;
```
AFAICS this is not initialized anywhere.

Reply:
Oops, indeed it wasn't initialized.